README.md

```diff
@@ -109,6 +109,114 @@ const inst = new Instance({
 ```
 
 
+
+#### Highlight
```

Comment:
How about adding an HTML highlight tutorial here? (with <!-- [highlight] -->)
I think it's good to show that we also support that.